src/scenes/AddEdit/recurrence-selector.jsx

```diff
@@ -127,27 +128,29 @@ class EndOptions extends React.Component {
 
 
   render(){
-    let untilDate = this.state.option === 'date' ? <EventDateTimeSelector datetime = {this.state.endDate} onChange = {this.endDateHandler}/> : null;
+    let untilDate = this.state.option === 'date' ? <EventDateTimeSelector datetime = {this.state.endDate} onChange = {this.endDateHandler} show="date"/> : null;
     return(
       <div className="radio-collection-container">
```

Comment:
Add another class: `recurrence-selector`. You can add a margin to it with CSS: `.recurrence-selector {margin-bottom: 1em;}`